yt/visualization/volume_rendering/scene.py

```diff
@@ -407,7 +447,8 @@ def save_annotated(
             suffix = ".png"
             fname = "%s%s" % (fname, suffix)
 
-        self.render()
+        self._render_if_missing(fname, render)
```

Comment:
this line is responsible for the failing tests: `fname` shouldn't be passed to this function anymore

Reply:
yup! caught that last night, but forgot to push up the commit to fix it. Will do that now...

Reply:
Ok, it's up. Includes the change to using `_sanitize_render`. Though I didn't make it a static function like you mentioned -- wasn't sure since it requires checking for `self._last_render`. If it were static, wouldn't the function be built on initialization when `self._last_render=None`?

Reply:
yeah I was clearly tired, I went back and reverted that suggestion on slack 😅